package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "tsc",
     "prepack": "yarn build",
-    "lint": "yarn eslint --ext .ts,.js,.mjs src",
+    "lint": "yarn eslint --ext .ts,.js,.mjs src test",
```

Comment:
We added tests about four months ago and forgot to add the folder to the linter.